src/ruleset/v3/ruleset.ts

```diff
@@ -24,5 +25,18 @@ export const v3CoreRuleset = {
         function: operationMessagesUnambiguity,
       },
     },
+    'asyncapi3-required-operation-channel-unambiguity': {
+      description: 'Required operation (under root channels) "channel" must reference to a required channel (under root channels).',
```

Comment:
```suggestion
      description: 'Operation channel's must reference to a required channel  (under root channels).',
```

Reply:
But this only applies to those operations defined as required (under root). So I still think we should clarify this. 
I agree my sentence is not so good.

Reply:
Answered in https://github.com/asyncapi/parser-js/pull/913/files#r1407464169